setup.py

```diff
@@ -178,7 +178,6 @@
             'hepnames2marc = inspirehep.dojson.hepnames2marc:hepnames2marc',
         ],
         'invenio_base.api_apps': [
-            'inspire_theme = inspirehep.modules.theme:INSPIRETheme',
```

Comment:
I'm not sure if this is correct, because some Jinja filters are run when sending stuff from the API... let's see what the test suite says.